javascript/builtins/Iterator.json

```diff
@@ -48,18 +48,37 @@
             "spec_url": "https://tc39.es/ecma262/multipage/control-abstraction-objects.html#sec-%iteratorprototype%-@@iterator",
             "support": {
               "chrome": {
-                "version_added": "38"
+                "version_added": false
```

Comment:
`Iterator` is not a global (yet). You need to use `Symbol.iterator in Object.getPrototypeOf(Object.getPrototypeOf([].values()))`.